pyspecProcScripts/load_data.py

```diff
@@ -682,9 +691,13 @@ def proc_spincore_generalproc_v1(s, include_tau_sub=True, fl=None):
         s.reorder("ph_overall")
     # }}}
     # {{{ apply the receiver response
-    s /= s.fromaxis("t2").run(
-        lambda x: np.sinc(x / (s.get_prop("acq_params")["SW_kHz"] * 1e3))
+    s.set_prop(
+        "dig_filter",
+        s.fromaxis(direct).run(
+            lambda x: np.sinc(x / (s.get_prop("acq_params")["SW_kHz"] * 1e3))
+        ),
     )
+    s /= s.get_prop("dig_filter")
```

Comment:
save digital filter as property in case we want to mess with it or access it in later steps